mlir/lib/Dialect/MIOpen/Transforms/CloneKernels.cpp

```diff
@@ -28,50 +28,65 @@
 #include "llvm/ADT/SmallVector.h"
 
 using namespace mlir;
+using namespace llvm;
```

Comment:
I think instead of this, there's a header you can include that reexports most of the common llvm bits, like StringRef, into MLIR. `using namespace llvm;` has caused too many cases of otherwise-unneeded prefixing, IMO.

Reply:
Yeah.  `mlir/include/mlir/Support/LLVM.h` So strong disapprove of `using namespace llvm;` - it causes a _lot_ of headaches.